nomahcoffee/settings.py

```diff
@@ -59,6 +61,13 @@
     'whitenoise.middleware.WhiteNoiseMiddleware',
 ]
 
+CORS_ORIGIN_ALLOW_ALL = True
+#ALTERNATIVE 
```

Comment:
Also I think it would be safer to go with the alternative option here. And do something like:

```
CORS_ORIGIN_ALLOW_ALL = False
CORS_ORIGIN_WHITELIST = (
    'nomah-coffee.herokuapp.com', 
    '127.0.0.1', 
    'nomahcoffee.com',
    'www.nomahcoffee.com'
)
```

Reply:
That is probably a better idea, allowing only specific sites.